src/smartpeak/source/core/WorkflowManager.cpp

```diff
@@ -57,6 +57,11 @@ namespace SmartPeak {
   void WorkflowManager::run_and_join(ApplicationHandler& application_handler, bool& done, ApplicationHandler& source_app_handler, const std::set<std::string>& injection_names, const std::set<std::string>& sequence_segment_names, const std::set<std::string>& sample_group_names, const std::vector<ApplicationHandler::Command>& commands)
   {
     // run workflow asynchronously
+    // making local copies
```

Comment:
I believe these should be removed.  Please check the develop branch.  I would probably recommend merging develop as well to check for any conflicts.

Reply:
Yes, thanks for reminding